_meeting-notes/10-10-2019.md

```diff
@@ -0,0 +1,27 @@
+# Sync Oct 10, 2019
+
+## Attendees
+
+- David Dias
+- Jim Pick
+
+## Notes
+
+- AI(David): create PR to list the current team and roles 
+  - Raul -> Architect 
+  - Jim
+  - David
+- AI(David): Review the 1st video demo by Jim
```

Comment:
Reviewing the video:

- Your voice gives the whole video an angelical feeling, makes it pleasant to see.
- The video spends a considerable amount of time explaining things that can be assumed that the team knows (it would be a good video if the project was at a more advanced stage and we wanted to bring people outside of the main team), for example:
  - Don't need to explain that we have a 1password vault and how to get there. Just say: fetch the account from 1password, we can expect for us to know how to get it :)
  - Then say: use the root account to set one account for yourself with 2FA, url here -> done. Let us figure out the steps (even better, just create the account for me and Raul and Steven)

Reply:
Also, I noticed that the total time spent creating the video was ~4 hours. Let's rather start with a document and screenshots that gives tutorials rather than going full out. You should be able to do it in less than 30~45 minutes saving a ton of your time and energy.

Reply:
Yah, I figured I'd save time by recording a short video. Only after did I realize that I couldn't distribute it without blurring the heck out of it.

The primary reason I recorded that video is that it's basically giving the user confidence that "it's okay" to go in and create a new IAM account as long as there is MFA setup and that's it's an officially sanctioned way of doing that (assuming they already have access to the vault). At this stage of the project, I'm struggling with the top-level documentation design of where to put documentation assets where they can be found to effectively communicate. Video has the nice property of being self-contained.